backend/main.py

```diff
@@ -1,8 +1,23 @@
-from fastapi import FastAPI
 import json
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
 
+# this is to allow CORS so that you can connect backend and frontend locally
+origins = ["http://localhost",
+    "http://localhost:8000",

```

Comment:
Please utilize `.env` to centralize and control for all these local development
https://dev.to/felixdusengimana/environment-variables-in-reactjs-139l

Reply:
used it on f7db7e534712bc34e0ad2a5ed0265e65656181a2